reactor-netty-incubator-quic/build.gradle

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2011-Present VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+apply plugin: 'io.spring.javadoc'
+apply plugin: 'biz.aQute.bnd.builder'
+
+version="$reactorNettyQuicVersion"
+group = 'io.projectreactor.netty.incubator'
+
+ext {
+	bndOptions = [
+			"Export-Package" : "reactor.netty.incubator.quic*;version=$osgiVersion;-noimport:=true",
+			"Import-Package" : "!javax.annotation,*",
+			"Bundle-Name" : "reactor-netty-incubator-quic",
+			"Bundle-SymbolicName" : "io.projectreactor.netty.incubator.reactor-netty-incubator-quic",
+			"Bundle-Version" : "$osgiVersion"
+	]
+}
+
+dependencies {
+	compile project(':reactor-netty-core')
```

Comment:
please use `implementation` and `api` in new build.gradle. `compile`/`testCompile` and the like have been removed in Gradle 7.x, to which we will have to switch soon enough.